### ISSUE TITLE
Spaces at start of formula with greek symbols result in bad .tex

### ISSUE THREAD

**Issue by RubendeBruin** [jupyter/nbconvert#1298]
Using the following in a markdown cell works fine in a notebook:
$ \Delta$

Note the SPACE between $ and \

![afbeelding](https://user-images.githubusercontent.com/34062862/84994332-d35cec00-b14a-11ea-85c6-454d792289e0.png)

![afbeelding](https://user-images.githubusercontent.com/34062862/84994304-ca6c1a80-b14a-11ea-953c-d63b210068d5.png)

But when converting to .tex it is converted to:

\$ \Delta\$ 

which is invalid latex. It should have been

\( \Delta\)

To reproduce: Run attached notebook

[delta.ipynb.txt](https://github.com/jupyter/nbconvert/files/4797135/delta.ipynb.txt)

**Nbconvert version:** 5.6.1 (installed from conda-forge via jupyerlab)

**Comment by mgeier** [comment]
Duplicate:

* https://github.com/jupyter/nbconvert/issues/275#issuecomment-251878919
* #478
* #686
* #749
* #1172

**Comment by MSeal** [comment]
Would love some more contributors to help fix these things :). Though I do realize the learning curve to get started in nbconvert is non-trivial.

**Comment by RubendeBruin** [comment]
@MSeal , I will give it a try.

**Comment by MSeal** [comment]
I really appreciate that! Thank you

**Comment by RubendeBruin** [comment]
Somebody mentioned a non-trivial learning curve....

I've been digging and I think that where it goes wrong is in the part where the ninja2 magic happens. 

Some hints here would be nice.

I expect that in the ninja template there is some kind of regex that will find the $.....$ and replace it with ((......))

There is a Template.environment.filters.strip_dollars function which either costs a lot of money or is supposed to do just that. But no idea where the source of this function is. Googling on "strip and dollars" is not getting me anything useful.

Ah, found it. It is not what I hoped it would be. Searching further....





The easiest is a quick-fix. Simply replace and $[space] with $

**Comment by RubendeBruin** [comment]
ok, if all else fails then read the docs...

![afbeelding](https://user-images.githubusercontent.com/34062862/85062808-6cbaeb00-b1a9-11ea-8294-9655cf74d4d2.png)

So it is actually for a very good reason.

But if I enter $20,000 and $30,000 in jupyter lab then it does get converted to a formula.

So it is just a mismatch of whatever does the parsing in jupyter lab and pandoc.

@MSeal, Would nbconvert be the place to trick the input of pandoc such that is behaves the same as the notebook?

**Comment by RubendeBruin** [comment]
![afbeelding](https://user-images.githubusercontent.com/34062862/85063166-071b2e80-b1aa-11ea-991f-85dc82fd7581.png)

**Comment by MSeal** [comment]
Yes you got it on the head I think. Having nbconvert tricking the input of pandoc is the right approach here to match notebook expectations. The pattern this is done in elsewhere is a little inconsistent, so as more of the pandoc prep actions get written we'll want to organize them better. But if there's a fix + a test the organization needed later on should be easy.

**Comment by t-makaro** [comment]
Late to the party, but yes, the pandoc ```tex_math_dollars``` is the root of the problem. It may be possible to right a custom pandoc extension to replace ```tex_math_dollors```, but I have not looked at all into how to do that. We would also need to verify some of the edge cases in Jupyter lab with how it renders.
